### PR TITLE
Fixing home links

### LINKED ISSUE #412: Issue Regarding - 404 Error linking from home page
**I want to...** 

- [ ] Propose an update to an existing design.
- [ ] Report a bug with the current documentation.
- [ ] Work with the team to build a new pattern for to be considered for inclusion in Helix.
- [ ] Other, explain below

**Tell us a little more about your request?**
- "Pattern Guidelines" links back to the home page
- "Sketch Resources" leads to a 404 error

screen shot 2017-12-07 at 3 57 11 pm

- Invision icon links back to home page
screen shot 2017-12-07 at 4 05 23 pm

**How would you like to collaborate with us on this issue?** 

- [ ] I’d like to communicate via GitHub
- [ ] I’d like to communicate via Slack
- [x] I’d like the Helix team to handle it from here

### PR DESCRIPTION
**Pull request by nimbinatus**
Fixes #412

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -15,8 +15,8 @@ customers happy.
     <p>Start designing with Helix components.</p>
     <div class="card-bottom">
       <ul>
-        <li><a href="{{site.url}}">Pattern Guidelines</a></li>
-        <li><a href="https://github.com/rackerlabs/helix/">Sketch Resources</a></li>
+        <li><a href="{{site.url}}/status.html">Pattern Guidelines</a></li>
+        <li><a href="{{site.cdn_url}}/sketch/helix_stickersheet_v0.1.sketch">Sketch Resources <hx-icon type="download"></hx-icon></a></li>
```
Comment (robb-romans) [inline index.md]: do we really want to version the sticker sheet via the link?
might be better to make this a permanent un-versioned link, version within the target

File: templates/_includes/overview-footer.html [hunk 2]
```diff
@@ -7,7 +7,7 @@ <h2>Get in the Loop.</h2>
         <a href="https://github.com/rackerlabs/design-system"><img src="assets/images/GitHub-Mark-120px-plus.png" alt="GitHub"/></a>
       </div>
       <div class="hxCol-2">
-        <a href="#"><img src="assets/images/InVision-icon-black.png" alt="Invision"/></a>
+        <a href="https://rackspace.invisionapp.com/d/main#/projects"><img src="assets/images/InVision-icon-black.png" alt="Invision"/></a>
```
Comment (robb-romans) [inline templates/_includes/overview-footer.html]: note: this link requires sign-in

Comment (nimbinatus) [reply templates/_includes/overview-footer.html]: Yep! I don't think we have an icon for that, unfortunately...

File: index.md (hunk 1 above)
Comment (nimbinatus) [reply index.md]: At the moment, that's what it's titled as. I think there was a discussion about that during one of the team meetings; personally, I prefer to check into version control and drop the version in the title, but that's me :)

**Comment by nimbinatus** [comment]
Drew approved hitting the merge button myself as he's on the road.

**Pull request merged**